shub/image/list.py

```diff
@@ -140,12 +143,17 @@ def raise_shub_image_info_error(error):
     if not isinstance(spiders_list, list):
         raise_shub_image_info_error('spiders section must be a list')
 
+    project_type = metadata.get('project_type')
```

Comment:
@jdemaeyer we added server-side validation that'll return an error if project_type is missing or invalid. is there any good reason to duplicate server-side validation in shub?